celery/extractions.py

```diff
@@ -14,13 +20,109 @@
 FONCIER_EXTRACTS_DIR = env.get('FONCIER_EXTRACTS_DIR', '/tmp')
 FONCIER_STATIC_DIR = env.get('FONCIER_STATIC_DIR')
 
+PG_CONNECT_STRING = env.get("PG_CONNECT_STRING")
+
 taskmanager = Celery('extractions',
                      broker=CELERY_BROKER_URL,
                      backend=CELERY_RESULT_BACKEND)
 
+
+def run_command(args):
+    """
+    Run command specified by args. If exit code is not 0 then full command line, STDOUT, STDERR are printed and an
+    Exception is raised
+    :param args: array of argument
+    :return: None
+    """
+    p = Popen(args, stdout=PIPE, stderr=PIPE)
+    p.wait()
+
+    if p.returncode != 0:
+        print("Commande : %s" % " ".join(args))
+        print("Exit code : %s" % p.returncode)
+        print("STDOUT : %s" % p.stdout.read().decode())
+        print("STDERR : %s" % p.stderr.read().decode())
+        raise Exception("Error running %s" % " ".join(args))
+
+
+def get_all_tables(conn, year):
+    """
+    List tables in schema foncier_<year> where year is the second argument
+    :param conn: a psycopg connection instance
+    :param year: numerix year to append to 'foncier_' to build schema name
+    :return: array of table name
+    """
+    cur = conn.cursor()
+    cur.execute("SELECT table_name FROM information_schema.tables WHERE table_schema = 'foncier_%s'" % year)
+    res = [res[0] for res in cur.fetchall()]
+    cur.close()
+    return res
+
+
+def export_schema_to_shapefile_or_mapfile(year, proj, output_dir, format, conn, pg_connect_string):
```

Comment:
Please rename to `export_schema_to_shapefile_or_mapinfo` (no mapfile involved here ;-)